pages.de/common/git-setup.md

```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Erstelle ein Git Repositorium in einem bestimmten Verzeichnis und committe alle Dateien.
```

Comment:
the other pages don't translate it either
```suggestion
> Erstelle ein Git Repository in einem bestimmten Verzeichnis und committe alle Dateien.
```

Reply:
Interesting... Why? 😄 🤔